hack/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.16-stretch
```

Comment:
@esequielvirtuoso why using the stretch version?

Reply:
It is set as the default version in [gootstrap](https://github.com/NeowayLabs/gootstrap/blob/master/template/dockerfile.go). 
I've used this code generator as a recommendation.
@LeoCBS Can you help us figure that out?